examples/oft-solana/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "types": ["jest"],
+    "lib": ["es2015"],
```

Comment:
very old, why?

Reply:
It's pretty much standard and more than enough, we're not using any features from `ES7` or above